third_party/python.bzl

```diff
@@ -71,3 +71,14 @@ def tensorboard_python_workspace():
       strip_prefix = "six-1.10.0",
       build_file = str(Label("//third_party:six.BUILD")),
   )
+
+  native.new_http_archive(
+      name = "org_pythonhosted_portpicker",
```

Comment:
Will do. I'm investigating why the new @org_pythonhosted_portpicker dependency doesn't get picked up by the new py_library and py_tests, leading to the ImportError.